source/assets/scripts/main.js

```diff
@@ -0,0 +1,101 @@
+// main.js
+
+const $ = (selector) => document.querySelector(selector);
+
+
+const featureRecipes = [
+  'https://introweb.tech/assets/json/ghostCookies.json',
+  'https://introweb.tech/assets/json/birthdayCake.json',
+  'https://introweb.tech/assets/json/chocolateChip.json',
+  'assets/recipes/meatball.json',
+  'assets/recipes/quickEasyPaste.json',
+  'assets/recipes/pumpkinMuffins.json'
+];
+
+let myRecipeArray = JSON.parse(localStorage.getItem('myRecipeArray'));
+
+
+const feaRecipeData = {};
+const favRecipeData = {};
+
+window.addEventListener('DOMContentLoaded', init);
+
+
+async function init() {
+  let fetchSuccessful = await fetchRecipes();
+  if (!fetchSuccessful) {
+    console.log('Recipe fetch unsuccessful');
+    return;
+  };
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/cse110-fa21-group16/cse110-fa21-group16/pullRequest?prid=8393580)
```suggestion
  }
```